name.abuchen.portfolio/src/name/abuchen/portfolio/datatransfer/pdf/SimpelPDFExtractor.java

```diff
@@ -122,15 +123,34 @@ private void addDividendeTransaction()
             });
 
         pdfTransaction
-                // Fondsname: Standortfonds Deutschland Datum des Ertrags: 21.12.2021
-                // WKN / ISIN: AT0000A1Z882 Turnus: jährlich
-                .section("name", "isin")
-                .match("^Fondsname: (?<name>.*) Datum .*$")
-                .match("^WKN \\/ ISIN: (?<isin>[\\w]{12}) .*$")
-                .assign((t, v) -> {
-                    v.put("currency", CurrencyUnit.EUR);
-                    t.setSecurity(getOrCreateSecurity(v));
-                })
+                     
+        .oneOf(
+                        // @formatter:off
+                        // Fondsname: Standortfonds Deutschland Datum des Ertrags: 21.12.2021
+                        // Fondsname: Standortfonds Österreich Datum des Ertrags: 21.12.2022
+                        // @formatter:off
+                        section -> section //
+                        .attributes("name") //
+                        .match("^Fondsname: (?<name>.*) Datum .*$") //
+                        .assign((t, v) -> type.getCurrentContext().put("fondsName", v.get("name"))) //
+                        , //
+                        // @formatter:off
+                        // Fondsname: Standortfonds Österreich
+                        // @formatter:on
+                        section -> section //
+                        .attributes("name") //
+                        .match("^Fondsname: (?<name>.*)$") //
+                        .assign((t, v) -> type.getCurrentContext().put("fondsName", v.get("name"))) //
+                        )
+
+                        // WKN / ISIN: AT0000A1Z882 Turnus: jährlich
+                        // WKN / ISIN: AT0000A1QA38 Turnus: jährlich
+                        // WKN/ISIN: AT0000A1QA38 Datum des Ertrags: 15.12.2023
+                        .section("isin").match("^.*WKN\\s?\\/\\s?ISIN: (?<isin>[\\w]{12}).*$").assign((t, v) -> {
+                            v.put("currency", CurrencyUnit.EUR);
+                            v.put("name", type.getCurrentContext().get("fondsName"));
+                            t.setSecurity(getOrCreateSecurity(v));
+                        })
```

Comment:
The security is always created with the following values, which are also available in the PDF debug.

1. the currency of the security
2. the ISIN or WKN

The security name and ticker are optional.